templates/pages/admin/log_viewer.html.twig

```diff
@@ -0,0 +1,177 @@
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% macro log_entry_tpl(log) %}
+    {% set log_entry_html %}
+        <div class="log_entry d-flex" id="{{ log['id'] }}">
+            {% if log['text']|split("\n")|length > 2 %}
+                <input type="checkbox" class="log_entry_toggle" id="more_{{ log['id'] }}">
+                <label for="more_{{ log['id'] }}" class="log_entry_toggle_lbl" role="button">&nbsp;</label>
+            {% else %}
+                <span class="log_entry_toggle_placeholder"></span>
+            {% endif %}
+            <label class="log_long_text" for="more_{{ log['id'] }}">
+                <a class="log_datetime badge me-1" href="#{{ log['id'] }}">
+                    {{ log['datetime']|formatted_datetime }}
+                </a>
+                <##log_text##>
+            </label>
+        </div>
+    {% endset %}
+    {{ log_entry_html|spaceless|replace({
+        '<##log_text##>': log['text']|trim,
+    })|raw }}
+{% endmacro %}
+
+{% set log_entries_html %}
+    {% for log_entry in log_entries %}{{ _self.log_entry_tpl(log_entry)|trim|raw }}{% endfor %}
+{% endset %}
+
+{% if only_content %}
+    {{ log_entries_html }}
+    <span class="animated-dots" id="auto-refresh-indicator"></span>
+{% else %}
+    <div class="log-viewer card">
+        <div class="btn-list justify-content-between card-header">
+            <span class="d-flex align-items-start">
+                <div class="input-group">
+                    <select class="form-select form-select-sm w-auto" id="change-file">
+                    {% for log_slug, log_file in log_files %}
+                        set
+                        <option value="{{ log_slug }}"
+                                {{ log_file == filename ? 'selected="selected"' : '' }}>
+                            {{ log_file }}
+                        </option>
+                    {% endfor %}
+                    </select>
+
+                    <a href="?action=download_log_file&amp;filename={{ filename }}"
+                    class="btn btn-sm btn-outline-secondary"
+                    title="{{ __("Download file") }}"
+                    data-bs-toggle="tooltip" data-bs-placement="bottom">
+                        <i class="ti ti-device-floppy"></i>
+                    </a>
+                </div>
+
+                <span class="input-icon ms-3">
+                    <input type="text" value="" class="form-control form-control-sm" id="filter_logs_input"
+                        placeholder="{{ __("filter logs") }}">
+                    <span class="input-icon-addon">
+                        <i class="ti ti-filter fa-xs"></i>
+                    </span>
+                </span>
+            </span>
+            <span class="d-inline-flex">
+                {% set limitdropdown = include('components/dropdown/limit.html.twig', {
```

Comment:
This dropdown currently redirects to an invalid page `/front/glpilist_limit=5` when changed. No `href` is set and it isn't in a tab, so the default behavior doesn't work. Also, I'm not sure the fluid_search check is needed as the page doesn't initialize a fluid search view.